### PR TITLE
Release/v8.6.1

### PR DESCRIPTION
**Pull request by AliAbdelfattah**
### What's done?
- Added `enabled` key to `Instabug.reproStepsMode` enum.
- Updated CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-## Master
+## v8.6.1 (2019-08-26)
```
Comment (HeshamMegid) [inline CHANGELOG.md]: This should be 8.6, not 8.6.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 
-* Bump native SDKs to 8.6.1
+* Bumps version to 8.6 to be in sync with other platforms.
```
Comment (HeshamMegid) [inline CHANGELOG.md]: I don't think we need this line.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "instabug-reactnative",
-  "version": "8.5.6", 
+  "version": "8.6.1",
```
Comment (HeshamMegid) [inline package.json]: Same comment about the version number.

File: CHANGELOG.md (hunk 2 above)
Comment (alyezz) [reply CHANGELOG.md]: Do we need to mention re-branding here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
-## Master
+## v8.6 (2019-08-26)
```
Comment (alyezz) [inline CHANGELOG.md]: The version number in RN cannot be a 2 digit number, this should be 8.6.0

**Pull request merged**